br/tests/_utils/run_lightning

```diff
@@ -31,4 +31,5 @@ tidb-lightning.test -test.coverprofile="$COV_DIR/cov.$TEST_NAME.$$.out" DEVEL \
     --sorted-kv-dir "$TEST_DIR/$TEST_NAME.sorted" \
     --enable-checkpoint=0 \
     --check-requirements=0 \
+    -L debug \
```

Comment:
after the debug is finished, I think we should delete this change. Maybe it will interfere other integration tests

Reply:
don't forget to remove this